Library/src/Systems/Sequence/SequenceSystem.cpp

```diff
@@ -127,6 +175,12 @@ void SequenceSystem::GetSequencesByCriteria(const Array<String>& InSequenceKeys,
 
 void SequenceSystem::GetSequence(const String& SequenceKey, SequenceResultCallback Callback)
 {
+	if (!ValidateKey(SequenceKey))
+	{
+		CSP_LOG_FORMAT(csp::systems::LogLevel::Error, "Cannot get Sequence. New Key: %s contains invalid characters", SequenceKey.c_str());
```

Comment:
I don't think the log message should say "New Key:", just "Key:" as it is not a new key.